lua/lualine/components/aerial.lua

```diff
@@ -0,0 +1,36 @@
+local M = require("lualine.component"):extend()
+local aerial = require("aerial")
+
+local default_options = {
+  sep = " ⟩ ",
```

Comment:
**n.b.** I can't call it `separator` since `separator` is reserved by `lualine`.